### PR TITLE
Dikhtiarenko_Karina

### PR DESCRIPTION
**Pull request by KarinaDikhtiarenko**

### CODE REVIEW

File: Lab1/test/lab5_1.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
+/*�������� 1
```
Comment (Polivando) [inline Lab1/test/lab5_1.java]: lol

File: Lab1/test/lab5_2.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*�������� 2
```
Comment (Polivando) [inline Lab1/test/lab5_2.java]: still lol

File: Lab1/test/lab5_3.java [hunk 3]
```diff
@@ -0,0 +1,304 @@
+/*�������� 3
```
Comment (Polivando) [inline Lab1/test/lab5_3.java]: now you know the pain of unsupported encodings

**Comment by KarinaDikhtiarenko** [comment]
Lol.
Now I know.

**Pull request merged**